### PR TITLE
fedora Dockerfile : Switch to f36 and OVN 22.06.

### PR DESCRIPTION
**Pull request by numansiddique**
Signed-off-by: Numan Siddique <numans@ovn.org>

**- What this PR does and why is it needed**

**- Special notes for reviewers**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by tssurya** [comment]
CI doesn't look good. Praying its not some OVN regression

**Review by flavio-fernandes — COMMENTED** [review]
/lgtm

File: dist/images/Dockerfile.fedora [hunk 1]
```diff
@@ -9,13 +9,13 @@
...
 
-FROM fedora:35
+FROM fedora:36
 
 USER root
 
 ENV PYTHONDONTWRITEBYTECODE yes
 
-ARG ovnver=ovn-22.06.0-2.fc35
+ARG ovnver=ovn-22.06.0-2.fc36
```
Comment (flavio-fernandes) [inline dist/images/Dockerfile.fedora:18]: @numansiddique is there a `22.07` for fc36 ?

**Comment by numansiddique** [comment]
@flavio-fernandes   This PR has 22.06.0-27 now

**Review by flavio-fernandes — COMMENTED** [review]
@numansiddique we may have some extra work to do here in order to make fc6 work with kind:

`F0801 14:31:36.067659    3832 ovnkube.go:133] no system-id configured in the local host`

system-id should have been configured:  

```
func GetNodeChassisID() (string, error) {
...
```

https://github.com/numansiddique/ovn-kubernetes/blob/b5948436f5785fb8440efc1b539445c0d2792863/go-controller/pkg/util/util.go#L53

**Comment by flavio-fernandes** [comment]
/lgtm
TY @numansiddique !

**Comment by flavio-fernandes** [comment]
k exec -it -n ovn-kubernetes       ovnkube-master-7f9fc84984-mprn6 -c ovnkube-master -- bash
```
[root@ovn-control-plane ~]# cat /etc/*relea*
Fedora release 36 (Thirty Six)
NAME="Fedora Linux"
VERSION="36 (Container Image)"
ID=fedora
VERSION_ID=36
...
```

```
[root@ovn-control-plane ~]# rpm -qa | grep -i ovn
ovn-22.06.0-27.fc36.x86_64
ovn-central-22.06.0-27.fc36.x86_64
ovn-host-22.06.0-27.fc36.x86_64
ovn-vtep-22.06.0-27.fc36.x86_64
```

**Comment by numansiddique** [comment]
@trozet Does this look good to you ?

**Review by trozet — APPROVED** [review]

**Pull request merged**